### PR TITLE
Add missing changes of PR 226 to the CHANGELOG

### PR DESCRIPTION
**Pull request by matthiasschaub**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,8 +17,10 @@
 - Fix concurrent execution of CLI function `create_all_indicators` using async and semaphores ([#254])
 - Support choosing a single indicator and/or single layer for CLI command `create_all_indicators` ([#254])
 - Indicators based on GHS-POP use raster file stored on disk instead of raster in the database ([#276])
+- Reverse [#221], rename `ideal_vgi_lulc` layer and return result description in the case that the Mapping Saturation Indicator could not been calculated ([#226])
 
 [#221]: https://github.com/GIScience/ohsome-quality-analyst/pull/221
+[#226]: https://github.com/GIScience/ohsome-quality-analyst/pull/260
```
Comment (joker234) [inline CHANGELOG.md:23]:
```suggestion
[#260]: https://github.com/GIScience/ohsome-quality-analyst/pull/260
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,8 +17,10 @@
 - Fix concurrent execution of CLI function `create_all_indicators` using async and semaphores ([#254])
 - Support choosing a single indicator and/or single layer for CLI command `create_all_indicators` ([#254])
 - Indicators based on GHS-POP use raster file stored on disk instead of raster in the database ([#276])
+- Reverse [#221], rename `ideal_vgi_lulc` layer and return result description in the case that the Mapping Saturation Indicator could not been calculated ([#226])
```
Comment (joker234) [inline CHANGELOG.md:20]:
```suggestion
- Reverse [#221], rename `ideal_vgi_lulc` layer and return result description in the case that the Mapping Saturation Indicator could not been calculated ([#260])
```

Comment (joker234) [inline CHANGELOG.md:20]: Did you maybe mean _revert_ instead of _reverse_?

Comment (matthiasschaub) [reply CHANGELOG.md:20]: How about this PR number?
```suggestion
- Reverse [#221], rename `ideal_vgi_lulc` layer and return result description in the case that the Mapping Saturation Indicator could not been calculated ([#286])
```

Comment (matthiasschaub) [reply CHANGELOG.md:20]: Oh sorry. Scratch my comment

Comment (matthiasschaub) [reply CHANGELOG.md]: I meant revert. Thanks.

**Review by joker234 — APPROVED** [review]

**Pull request merged**